Source/Core/Core/Config/GraphicsSettings.cpp

```diff
@@ -69,17 +69,10 @@ const Info<bool> GFX_BORDERLESS_FULLSCREEN{{System::GFX, "Settings", "Borderless
 const Info<bool> GFX_ENABLE_VALIDATION_LAYER{{System::GFX, "Settings", "EnableValidationLayer"},
                                              false};
 
-#if defined(ANDROID)
-const Info<bool> GFX_BACKEND_MULTITHREADING{{System::GFX, "Settings", "BackendMultithreading"},
-                                            false};
-const Info<int> GFX_COMMAND_BUFFER_EXECUTE_INTERVAL{
-    {System::GFX, "Settings", "CommandBufferExecuteInterval"}, 0};
-#else
 const Info<bool> GFX_BACKEND_MULTITHREADING{{System::GFX, "Settings", "BackendMultithreading"},
                                             true};
 const Info<int> GFX_COMMAND_BUFFER_EXECUTE_INTERVAL{
     {System::GFX, "Settings", "CommandBufferExecuteInterval"}, 100};
```

Comment:
This `GFX_COMMAND_BUFFER_EXECUTE_INTERVAL` setting doesn't seem to be exposed anywhere. Apparently this difference was relevant on Mali in 2019: #8379.

Reply:
It's not obvious why he did that change. It has nothing to do with slow memory types which is what that PR was about.

Reply:
FWIW, that option was problematic before because it would just end up waiting for previous submissions. My submit PR that was merged earlier fixed that.